custom_components/nhc2/nhccoco/devices/device.py

```diff
@@ -1,3 +1,5 @@
+from typing import Union, List
```

Comment:
Why this `List` and not `list`. 
I'm not a python expert, and throughout the codebase `list` is used. Can you change this for the sake of having consitency